src/sage/categories/finite_dimensional_lie_algebras_with_basis.py

```diff
@@ -1068,7 +1068,7 @@ def is_semisimple(self):
             return not self.killing_form_matrix().is_singular()
 
         @cached_method(key=lambda self,M,d,s,n: (M,d,s))
-        def chevalley_eilenberg_complex(self, M=None, dual=False, sparse=True, ncpus=None):
+        def chevalley_eilenberg_complex(self, M, h, dual=False, sparse=True, ncpus=None):
```

Comment:
You are making multiple backwards incompatible changes. You should leave the option that `M=None` (and subsequently add `h=None`) for the trivial module.